### ISSUE TITLE
Issue of nan Reconstruction loss when hidden layer is of dimension 1000

### ISSUE THREAD

**Issue by ayushgarg1804** [blackecho/Deep-Learning-TensorFlow#62]
When I run the following command from command line, I get reconstruction loss of nan
>  python run_deep_autoencoder.py --rbm_layers 1000 --batch_size 10 --rbm_num_epochs 30 --verbose 1 --finetune_learning_rate 0.1 --finetune_num_epochs 50 --opt momentum --learning_rate 0.05 --enc_act_func sigmoid --dec_act_func sigmoid --loss_func mse --finetune_act_func sigmoid --rbm_noise gauss

I noticed this only occurs if hidden layer dimensions are more than visible feature dimensions.
